src/components/views/elements/PowerSelector.tsx

```diff
@@ -60,6 +60,7 @@ export default class PowerSelector<K extends undefined | string> extends React.C
         maxValue: Infinity,
         usersDefault: 0,
     };
+    public unmounted = false;
```

Comment:
can't this be private?

```suggestion
    private unmounted = false;
```

Reply:
Oh yup